styles.css

```diff
@@ -1,3 +0,0 @@
-.hidden {
-  display: none !important;
```

Comment:
There are no longer global styles and I'm no longer adding this custom class to the class list of any element, so removed this file as well.